docs/expressions.md

```diff
@@ -0,0 +1,78 @@
+# Expressions
+
+Tracetest allows you to add expressions when writing your tests. They are a nice and clean way of adding values that are only known during execution time. For example, referencing a variable, a span attribute or even arithmetic operations.
+
+## Features
+
+* Reference span attributes
+* Arithmetic operations
+* String interpolation
+* Filters
+
+### Reference span attributes
+
+When building assertions, you will probably need to assert if a certain span contains an attribute and this attribute has a specific value. For this, you can use expressions to get the value of the span. When referencing an attribute, you must add the prefix `attr:` and its name. For example, imagine you have to check if the attribute `service.name` is equal to `cart-api`. You will have to use the following statement:
+
+```
+attr:service.name = "cart-api"
+```
+
+### Arithmetic operations
+
+Sometimes we need to manipulate data to ensure our test data is correct. Imagine a purchase operation. Now think how you would make sure that after the purchace, the product stock is smaller than before? For this, we might want to use arithmetic operations:
+
+```
+attr:product.stock = attr:product.stok_before_purchase - attr:product.number_bought_items
+```
+
+### String interpolation
+
+Some tests might require strings to be compared, but maybe you might need to generate a dynamic string that relies on some dynamic value. This might be used in an assertion, or even in the request body referencing some environment variable.
```

Comment:
```suggestion
Some tests might require strings to be compared, but maybe you need to generate a dynamic string that relies on a dynamic value. This might be used in an assertion or even in the request body referencing an environment variable.
```